v2-src/packet_processor.py

```diff
@@ -189,8 +189,8 @@ def _process_dns(self, pkt):
             return
 
         # Remove trailing dot from domain
-        if domain.endswith('.'):
-            domain = domain[0:-1]
+        if domain[-1] == '.':
+            main = domain[0:-1]
```

Comment:
@kumarde mmm unused variable. typo?

Reply:
Haha, whoops. It should be DOmain, not sure how that happened.